internal/utils.go

```diff
@@ -245,6 +270,28 @@ func setNamespaceSelector(policyConf *types.PolicyConfig, policyTemplate *map[st
 	}
 }
 
+// processKustomizeDir runs a provided directory through Kustomize in order to generate the manifests within it.
+func processKustomizeDir(path string) (*[]map[string]interface{}, error) {
+	k := krusty.MakeKustomizer(krusty.MakeDefaultOptions())
+
+	resourceMap, err := k.Run(filesys.MakeFsOnDisk(), path)
+	if err != nil {
+		return nil, fmt.Errorf("failed to process provided kustomize directory: %w", err)
```

Comment:
It does, but it would be good to put it up front, especially since it can give quite a trace:
```
error generating policies from the PolicyGenerator file '/var/folders/mt/6766yzb50jlb9t0x6gvl0nz00000gn/T/kust-plugin-config-184116644': failed to process provided kustomize directory: accumulating resources: accumulation err='accumulating resources from 'dev': '/Users/dhaiduce/github/open-cluster-management/grc/extras/policy-generator-plugin/examples/input-kustomize/dev' must resolve to a file': recursed accumulation of path '/Users/dhaiduce/github/open-cluster-management/grc/extras/policy-generator-plugin/examples/input-kustomize/dev': accumulating resources: accumulation err='accumulating resources from '../base': '/Users/dhaiduce/github/open-cluster-management/grc/extras/policy-generator-plugin/examples/input-kustomize/base' must resolve to a file': recursed accumulation of path '/Users/dhaiduce/github/open-cluster-management/grc/extras/policy-generator-plugin/examples/input-kustomize/base': accumulating resources: accumulation err='accumulating resources from 'nginx-pod.yaml': yaml: line 8: mapping values are not allowed in this context': got file 'nginx-pod.yaml', but '/Users/dhaiduce/github/open-cluster-management/grc/extras/policy-generator-plugin/examples/input-kustomize/base/nginx-pod.yaml' must be a directory to be a root
```

Reply:
Side note: It'd be nice if we could figure out how to reference the original file instead of the tmp generator file. That might not be possible, though.